<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DomoticaQTApp</class>
 <widget class="QWidget" name="DomoticaQTApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DomoticaQTApp</string>
  </property>
  <widget class="QListWidget" name="nodeList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="redLed">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Red LED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="greenLed">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Green LED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAssign">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Assign button</string>
   </property>
  </widget>
  <widget class="QListWidget" name="buttonSelectList">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="debugWindow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
